ru/mail/chrome/communicator/wallet/SignonViewer.dtd

```diff
@@ -1,14 +1,14 @@
-<!ENTITY      windowtitle.label               "Менеджер паролей">
+﻿<!ENTITY      windowtitle.label               "Менеджер паролей">
 <!ENTITY      windowalttitle.label            "Менеджер форм: Сайты">
-<!ENTITY      tab.signonsstored.label         "Сохранённые пароли">
-<!ENTITY      tab.signonsnotstored.label      "Пароли никогда не будут сохраняться">
-<!ENTITY      tab.nopreview.label             "Предварительно введённые в форму данные никогда не будут сохраняться">
-<!ENTITY      tab.nocapture.label             "Введённые в форму данные никогда не будут сохраняться">
+<!ENTITY      tab.signonsstored.label         "Сохраненные пароли">
```

Comment:
CORRECTIONS NEEDED, line 3 should look like -<!ENTITY      tab.signonsstored.label         "Сохранённые пароли">. I propose to keep using "ё" without exceptions, since it's already widely used.